AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -7,6 +7,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+news.sky.com###uz_popup_container
```

Comment:
What the purpose of this rule?

Reply:
To hide a survey request.

Reply:
![Screen4343](https://user-images.githubusercontent.com/98787049/203546968-6f4682ba-9b4d-4b83-87c8-ba0891d1da75.png)